src/app/clients/deleteClient.ts

```diff
@@ -14,6 +15,8 @@ const deleteClient = async (clientId: string): Promise<DeleteClientResponse> =>
         clientid: clientId,
     });
 
+    getClientParcelsDetails(clientId);
```

Comment:
Hi @mtharkins I've just started looking at the outstanding PRs.

I _think_ that this isn't needed - you've made the necessary change in ClientsPage.tsx and I don't think this line does anything. I'm guessing that this was an earlier attempt that you changed your mind about.

Please let me know if I've misunderstood, but for now I'll revert this file.